bin/configs/java-native-8.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: java
+outputDir: samples/openapi3/client/petstore/java/native-java8
```

Comment:
I think `native` requires JDK11+ so I would remove `java8` as it's a bit confusing.

Reply:
updated as suggested in https://github.com/OpenAPITools/openapi-generator/commit/0ddb2a9754c3085dd0f14da7a9b42982965620fb